app/main/views/buyers.py

```diff
@@ -28,6 +27,7 @@
 from odf.style import TextProperties, TableRowProperties, TableColumnProperties, TableCellProperties, FontFace
 
 from io import BytesIO
+from six import text_type
```

Comment:
I still maintain we don't even need this, now we're definitively (*except for scripts) a P3K shop. @allait will comment if I misunderstood something :)

Reply:
@galund In terms of this being a purely flake8 PR I'd say allow this in order to change as little functionality as possible.
I'd like to see someone go through all the repos and remove _all_ the py2-3 compat stuff at once and we can officially drop py2.

Reply:
FWIW I think we should be doing that as we go, I can't imagine a product person prioritising a ticket that said "go through all the repos" doing _anything_ much ;-)

Reply:
Well I probably do have a spare hour today 😸

Reply:
@galund sure we _should_ but we wont. We did get tickets in to flake8 all the repos to be fair. Just needs someone pushing hard enough.

Reply:
Firebreak sprint? ;)

Reply:
I think it's reasonable to have 'drop Py2 support' in a separate PR, so we have more control over when we release it?

Reply:
Absolutely, doesn't have to be here, up to you. But we already dropped support, so this is just about removing redundant code, as and when it's convenient.